README.md

```diff
@@ -50,15 +50,27 @@ docker run -it --rm -v /dev:/dev --privileged --net=host microros/micro-ros-agen
 ```
 
 ## What files are relevant?
-- `pico_ros.c`: Contains the board specific implementation of the serial transport (no change needed).
+- `pico_uart_transport.c`: Contains the board specific implementation of the serial transport (no change needed).
 - `CMakeLists.txt`: CMake file.
 - `pico_micro_ros_example.c`: The actual ROS 2 publisher.
 
-## How Micro-ROS is compiled?
+## How to build the precompiled library
 
-Micro-ROS is precompiled for Raspberry Pi Pico in [`uros_ws/firmware/build`](uros_ws/firmware/build).
-If you want to compile it by yourself then check the [`uros_ws`](uros_ws) directory.
+Micro-ROS is precompiled for Raspberry Pi Pico in [`libmicroros`](libmicroros).
+If you want to compile it by yourself:
 
+<!-- 
+pushd extras/library_generation
+docker build . -t microros/micro_ros_arduino_builder:foxy
+popd
+ -->
+
+```bash
+docker pull microros/micro_ros_arduino_builder:foxy
```

Comment:
```suggestion
```
This is not needed, the line below automatically pulls the image.

Reply:
I'm going to keep it to ensure users to have the latest Dockerhub image, if an outdated local image exists, `docker run` does not update it.